packages/core/src/Dropdown/Dropdown.test.tsx

```diff
@@ -177,4 +177,34 @@ describe("Dropdown", () => {
     await userEvent.click(getCheckBox());
     expect(getApplyButton()).toBeDisabled();
   });
+
+  it.only("should not select disabled items when Select All is clicked", async () => {
```

Comment:
[`.only`](https://vitest.dev/guide/filtering) disables other tests
```suggestion
  it("doesn't select disabled items when Select All is clicked", async () => {
```

Reply:
Sorry, left that behind by mistake. Fixed.